prov/rxd/src/rxd_attr.c

```diff
@@ -63,7 +63,11 @@ struct fi_ep_attr rxd_ep_attr = {
 	.protocol_version = 1,
 	.max_msg_size = SIZE_MAX,
 	.tx_ctx_cnt = 1,
-	.rx_ctx_cnt = 1
+	.rx_ctx_cnt = 1,
+	.max_order_raw_size = SIZE_MAX,
+	.max_order_war_size = SIZE_MAX,
```

Comment:
Is WAR ordering guaranteed?  @aingerson can you confirm the supported data ordering semantics?

Reply:
No, WAW and RAW are guaranteed but WAR ordering is not.

Reply:
@shefty  @aingerson Thanks! Fixed